jupyterlab_server/settings_handler.py

```diff
@@ -35,7 +35,14 @@ def get(self, schema_name=""):
         """Get setting(s)"""
         # Need to be update here as translator locale is not change when a new locale is put
         # from frontend
-        translator.set_locale(self.get_current_locale())
+        try:
+            locale = self.get_current_locale()
+        except web.HTTPError as e:
+            # fallback in case of missing (404) or misshapen (500) translation schema
+            locale = DEFAULT_LOCALE
+            'Failed loading or validating translation settings schema'
```

Comment:
@telamonian Should this be wrapped in `self.log.warn`? Is it needed even with the handling of the error in `get_current_locale`?